internal/dag/gatewayapi_processor.go

```diff
@@ -165,42 +248,24 @@ func selectorMatches(selector metav1.LabelSelector, objLabels map[string]string)
 	return true, nil
 }
 
-func (p *GatewayAPIProcessor) computeHTTPRoute(route *gatewayapi_v1alpha1.HTTPRoute) {
+func (p *GatewayAPIProcessor) computeHTTPRoute(route *gatewayapi_v1alpha1.HTTPRoute, listenerSecret *Secret) {
 	routeAccessor, commit := p.dag.StatusCache.HTTPRouteAccessor(route)
 	defer commit()
 
-	// Validate TLS Configuration
-	if route.Spec.TLS != nil {
-		routeAccessor.AddCondition(status.ConditionNotImplemented, metav1.ConditionTrue, status.ReasonNotImplemented, "HTTPRoute.Spec.TLS: Not yet implemented.")
+	hosts, errs := p.computeHosts(route)
+	for _, err := range errs {
+		routeAccessor.AddCondition(status.ConditionResolvedRefs, metav1.ConditionFalse, status.ReasonDegraded, err.Error())
```

Comment:
Worth adding status tests for this and the other status being set in this helper?

Reply:
yup good call, added some more.